ci-operator/step-registry/ipi/conf/user-workload-monitoring/ipi-conf-user-workload-monitoring-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Use yq to create cluster monitoring config, as other steps may adjust it
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
```

Comment:
nit: Can we use JSON / `jq` for the manifest, so we don't have to `curl`-install `yq`?

Reply:
I guess that would hamper the readability - and we've already settled on yaml manifests in most steps. Some other steps are pulling `yq` too, so I guess its "canon" now :)